icicle/backend/cpu/src/hash/cpu_keccak.cpp

```diff
@@ -6,15 +6,19 @@ namespace icicle {
   class KeccakBackend : public HashBackend
   {
   public:
-    KeccakBackend(uint64_t input_chunk_size, uint64_t output_size, uint64_t rate, int padding_const)
-        : HashBackend(output_size, input_chunk_size)
+    KeccakBackend(uint64_t input_chunk_size, uint64_t output_size, uint64_t rate, int padding_const, const char* name)
+        : HashBackend(name, output_size, input_chunk_size)
     {
     }
 
     eIcicleError hash(const std::byte* input, uint64_t size, const HashConfig& config, std::byte* output) const override
     {
-      ICICLE_LOG_INFO << "Keccak CPU hash() called";
-      // TODO implement
+      ICICLE_LOG_DEBUG << "Keccak/sha3 CPU hash() called, batch=" << config.batch
+                       << ", single_output_size=" << output_size();
+      // TODO implement real logic
```

Comment:
prefer to add asserion

Reply:
yes it's just that it will fail in CI. Aviad has a PR for that already.